docs/reference/migration/migrate_5_0/suggest.asciidoc

```diff
@@ -25,35 +25,22 @@ to suggestion entries for both context and completion suggesters.
 
 ==== Completion suggester is document-oriented
 
-Suggestions are aware of the document they belong to. This enables
-retrieving any field value from the document. This is exposed
-through the query-time `payload` option in `completion` and `context`
-suggesters:
-
-[source,sh]
----------------
-GET /my_index/_search
-{
-  "suggest": {
-    "fooSuggestion": {
-       "text": "f"
-       "completion": {
-         "field": "fooSuggest",
-         "payload": ["field1", "field2"]
-       }
-     }
-  }
-}
----------------
+Suggestions are aware of the document they belong to. Now, associated
```

Comment:
Perhaps we should rephrase this with a "version 5.0" banner (I don't actually know how to make that in AsciiDoc, so maybe you can find it before I find an example).

> New in 5.0: Completion suggestions no longer need or support a `payload`. Instead, the suggestion is associated with the `_source` document, which means that the entire document will be returned instead of a custom payload.
> WARNING: In order to return the `_source` document, the `_source` meta-field must be enabled, which is the default behavior. If it is disabled, then 

And since 2.x-based indices behave differently, we need to keep a note about it:

> In 2.x: For indices created in 2.x, the completion suggester will still return the defined `payload`. (**keep above example**)